<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLuminousCalibrateItemWidget</class>
 <widget class="QWidget" name="HLuminousCalibrateItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="8">
    <widget class="HEntireTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_001">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>当前采样值：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_1</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>24</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_002">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>当前测试值：</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_2</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_003">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;font color=#000000&gt;cd/m&lt;sup&gt;2&lt;/sup&gt;&lt;/font&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HEntireTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">HePlugin/HEntireTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
